### PR TITLE
Run `cargo fmt`

### PR DESCRIPTION
**Pull request by fitzgen**
It's been a while, and we have a lot of reformatting with the latest version of `rustfmt`.

### CODE REVIEW

File: src/codegen/mod.rs [hunk 1]
```diff
@@ -1012,10 +1020,12 @@ impl CodeGenerator for CompInfo {
 
             // Try to catch a bitfield contination early.
             if let (Some(ref mut bitfield_width), Some(width)) =
-                   (current_bitfield_width, field.bitfield()) {
+                (current_bitfield_width, field.bitfield()) {
```
Comment (emilio) [inline src/codegen/mod.rs]: not a fan of this one :(

Comment (fitzgen) [inline src/codegen/mod.rs]: I often wouldn't make the specific choices `rustfmt` makes, but that's not really the point ;) The point is not having to make the choices, and argue about them with each other.

**Comment by fitzgen** [comment]
> I'm afraid this will cause a lot of unneeded pain in #468

Woops :(

I just wanted to get this stuff out from cluttering up my own feature branch :-/

**Comment by fitzgen** [comment]
Maybe we should revisit the decision not to fail CI on unformatted code?

**Comment by emilio** [comment]
You mean to fail CI on unformatted code? Yeah, that's probably fine, and avoids these noisy diffs.

**Comment by fitzgen** [comment]
> You mean to fail CI on unformatted code? Yeah, that's probably fine, and avoids these noisy diffs.

Ok, will send a new PR for this soon.

**Pull request merged**